hypervisor/src/lib.rs

```diff
@@ -62,5 +62,5 @@ pub fn new() -> std::result::Result<Arc<dyn Hypervisor>, HypervisorError> {
 
     Ok(Arc::new(hv))
 }
-#[cfg(all(feature = "mshv", target_arch = "x86_64"))]
+
```

Comment:
Unrelated change to this particular patch.

(Not sure how this will show up in GitHub UI)

Reply:
Also, there is an extraneous blank line between the two SoB lines.